packages/back-end/src/routers/saved-group/saved-group.controller.ts

```diff
@@ -53,7 +53,7 @@ export const getSavedGroups = async (
   const context = getContextFromReq(req);
   const { org } = context;
 
-  if (!context.permissions.canCreateSavedGroup()) {
+  if (!context.permissions.canCreateSavedGroup({ projects: [] })) {
     context.permissions.throwPermissionError();
   }
```

Comment:
I can't find any use of this in our code. Seems like we just get all saved groups through `/organization/definitions`. Do we want to get rid of this unused endpoint?